examples/green-house-migration/routes/api/metadata.py

```
@@ -0,0 +1,119 @@
+
+
+@router.get("/officis")
+def list_officis(pagination: Dict[str, Any] = Depends(pagination_dependency)):
+    """List all officis with pagination and metadata"""
+    return paginated_get("officis", pagination)
+
+
+@router.get("/custom_fields/{field_type}")
+def list_custom_fields_by_type(
+    field_type: str,
+    pagination: Dict[str, Any] = Depends(pagination_dependency),
```

Ruff (0.12.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)